build.gradle

```diff
@@ -95,6 +95,10 @@ dependencies {
 	annotationProcessor group: "org.springframework.boot", name: "spring-boot-configuration-processor"
 }
 
+dependencyLocking {
+	lockAllConfigurations()
```

Comment:
If there's any really good reference for this, it might be nice to link somewhere.

Reply:
Yeah, it's https://docs.gradle.org/current/userguide/dependency_locking.html, I'll add a comment & put it in the readme

Reply:
+1 to adding info about dependency locking, and what to do if you need to change a dep version, to the README

Reply:
added to readme!